index.html

```diff
@@ -0,0 +1,103 @@
+                </h1>
+              </a>
+            </div>
+            <ul class="nav__links m-x-md">
+              <li class="m-x-sm p-x-sm">
+                <a class="btn btn-link p-b-lg" href="">Men</a>
+              </li>
+              <li class="m-x-sm p-x-sm">
+                <a class="btn btn-link p-b-sm" href="">Women</a>
+              </li>
+              <li class="m-x-sm p-x-sm">
+                <a class="btn btn-link p-b-sm" href="">Children</a>
+              </li>
+            </ul>
+            <div class="nav__searchbar m-x-sm">
+                <input
+                class="search-input border-rounded-md shadow-sm"
+                placeholder="Search..."
+                type="text"
+                id="text-input"
+              />
+              <i class="fas fa-search search-input__btn"></i>
+            </div>
+            <div class="nav__icons text-center">
+              <div class="nav__icons-cart">
+                <div class="badge__container text-center">
+                  <a href="./cart.html"
+                    ><img
+                      class="badge__container-img"
+                      src="./assets/cart.svg"
+                      alt="Cart-img"
+                    />
+                    <div
+                      class="badge badge-top-right border-rounded-full bg-primary text-offwhite bold-xl"
+                    >
+                      10
+                    </div>
+                    <span class="text-5 text-primary bold-lg center-x"
+                      >Cart</span
+                    ></a
+                  >
+                </div>
+              </div>
+              <div class="nav__icons-wishlist">
+                <div class="badge__container text-center">
+                  <a href="./cart.html"
+                    ><img
+                      class="badge__container-img"
+                      src="./assets/wishlist.svg"
+                      alt="Cart-img"
+                    />
+                    <div
+                      class="badge badge-top-right border-rounded-full bg-primary text-offwhite bold-xl"
+                    >
+                      10
+                    </div>
+                    <span class="text-5 text-primary bold-lg"
+                      >Wishlist</span
+                    ></a
+                  >
+                </div>
+              </div>
+              <div class="nav__icons-login">
+                <a href="./login.html" class="btn btn-secondary btn-icon border-rounded-md bg-light"><i class="fas fa-user btn__icon"></i>Login</a>
+                </div>
+              </div>
+            </div>
+          </nav>
+        </header>
+        <div class="main-wrapper">
+          <main class="main-container">
+            
+          </main>
+        </div>
+      </div>
+    </div>
+
+    <script
+      src="https://kit.fontawesome.com/fde0352739.js"
+      crossorigin="anonymous"
+    ></script>
```

Comment:
Use cdn in head to load external libraries.

Reply:
Added CDN to head tag.